GDJS/Runtime/pixi-renderers/CustomObjectPixiRenderer.ts

```diff
@@ -67,11 +67,12 @@ namespace gdjs {
       this._pixiContainer.pivot.x = this._object.getUnscaledCenterX();
       this._pixiContainer.pivot.y = this._object.getUnscaledCenterY();
       this._pixiContainer.position.x =
-        this._object.getDrawableX() +
+        this._object.getX() +
         this._pixiContainer.pivot.x * Math.abs(this._object._scaleX);
       this._pixiContainer.position.y =
-        this._object.getDrawableY() +
+        this._object.getY() +
         this._pixiContainer.pivot.y * Math.abs(this._object._scaleY);
```

Comment:
This was compensating the side effects of the camera bug.

The internal camera of custom objects was trying to stay at the center of the children. This way the transformation has no effect and children were not shifted. This system has been removed to avoid any side effects as it doesn't really make sense.